Polyhedron/demo/Polyhedron/Scene_c3t3_item.h

```diff
@@ -149,6 +149,8 @@ using namespace CGAL::Three;
 
   QColor get_histogram_color(const double v) const;
 
+  void itemAboutToBeDestroyed(Scene_item *);
```

Comment:
You should add the `Q_OVERRIDE` macro to it (I have not verified the precise name).